cypress/features/regression/accessibility/accessibilityCommon.feature

```diff
@@ -13,6 +13,14 @@ Feature: Accessibility tests
       | alert     |
       | sidebar   |
 
+  @ignore
+  # ignored because of accessibility issues after
+  # changing state of components -> https://jira.sage.com/browse/FE-2894
+  Scenario: Component button as sibling
```

Comment:
Is it a proper name of scenario for these steps?

Reply:
updated